dgbpy/dgbtorch.py

```diff
@@ -48,7 +87,8 @@ def hasTorch():
     'epochdrop': 5,
     'split': 0.2,
     'nbfold': 5,
-    'criterion': nn.CrossEntropyLoss() if hasTorch() else None,
+    'criterion': 'cross_entropy' if hasTorch() else None,
```

Comment:
@hpratama ,
the reason why the test is failing is because dgbpy expects a loss instance class for the criterion instead of a string. this is so that users can easily use/add their own custom loss functions